samples_project/Assets/SampleViewer/Scripts/SampleSwitcher.cs

```diff
@@ -7,24 +7,49 @@
 using Esri.ArcGISMapsSDK.Components;
 using Esri.HPFramework;
 using System.Collections;
-using System.Collections.Generic;
-using System.IO;
 using UnityEngine;
 using UnityEngine.Rendering;
 using UnityEngine.SceneManagement;
 using UnityEngine.UI;
 
 public class SampleSwitcher : MonoBehaviour
 {
-    public string APIKey = "";
-    public Dropdown PipelineTypeDropdown;
-    public Dropdown SceneDropdown;
+    public GameObject MenuVideo; 
```

Comment:
This doesn't look like you need this since you're only using it once in `StopVideo()`